bin/refresh-styles

```diff
@@ -4,11 +4,13 @@
 
 source bin/lib.sh
 
+bin/pull-image
+
 docker run -it --rm \
   -v /var/run/docker.sock:/var/run/docker.sock \
   -v $(pwd)/server:/usr/src/server \
   --entrypoint npx \
-  civiform/civiform-dev:latest \
+  civiform-dev \
```

Comment:
Can you help me understand this better?

I don't quite understand what the left is using or what this is now using.

Reply:
These are both tags.   "civiform/civiform-dev:latest" is the default tag, that lets it know to go to docker-hub and download the image if it's not cached locally.   In bin/pull-image, we add an additional tag of "civiform-dev" to the  "civiform/civiform-dev:latest" image. 

This is just a short-cut to make this script align better with the others, as well as allow someone to re-build locally and tag the build image.